feat: Implement full admin panel authorization with provider, model, …

EzequielDM opened the pull request:
…and user token management using SQLite storage.

god bless gemini 3.0 pro

vibe coded 😎

EzequielDM committed:
feat: Implement full admin panel authorization with provider, model, and user token management using SQLite storage.

god bless gemini 3.0 pro

Keramis reviewed (commented):
Upon creating a provider, it doesn't update the panel until a reload. Deletion works, though.


https://github.com/user-attachments/assets/3ca94dc4-7976-4e1b-a90f-f155fe9d1991

Keramis committed:
Fixed bugs

- `npm audit fix` (memory leak vuln)
- Fixed admin provider panel not refreshing after adding a provider

Keramis commented:
Fixed admin panel providers not refreshing in most recent commit

Keramis committed:
Added sessions to gitignore (admin sessinos)

Keramis committed:
Cleared session db

EzequielDM commented:
i love sessinos

Keramis commented:
"Check models" button doesn't activate upon adding the provider; needs to be, either:
- activated
- automatically fetch models upon adding


https://github.com/user-attachments/assets/e6a5a4ac-38ed-4c03-acd2-42323f47205d

Keramis committed:
Tried to add sessions to gitignore AGAIN (no clue why this doesn't detect)

Keramis committed:
Stop tracking sessions.sqlite

Keramis committed:
Merge branch 'sayori.chr' into fix-admin-panel

Keramis reviewed (approved):
Looks good!

Keramis commented:
@CrushedAsian255 Can you please review & verify before we pull?

Keramis commented:
<img width="426" height="57" alt="image" src="https://github.com/user-attachments/assets/05314220-49f1-4da1-89fa-f230032cec53" />
